tests/test_guard_diagnostics.py

```
@@ -44,6 +50,115 @@ def test_with_audit_trace(self):
+class TestIamGuardToVerificationContext:
+    def _verified_diagnostic(self):
+        result = VerificationResult(verified=True, allowed=True, proof="Z3 sat")
+        return IamGuard.to_diagnostic(result)
+
+    def test_verified_with_attestation(self):
+        guard = IamGuard()
+        diagnostic = self._verified_diagnostic()
+        vc = guard.to_verification_context(
+            diagnostic,
+            formal_statement="IAM policy is safe to apply",
+            attestation_token="fake-jwt",
```

Ruff (0.16.1)

[error] Possible hardcoded password assigned to argument: "attestation_token"

(S106)